package.json

```diff
@@ -41,7 +41,7 @@
     "ember-cli-htmlbars": "^3.0.1",
     "ember-cli-shims": "^1.2.0",
     "ember-click-outside": "^1.2.0",
-    "ember-computed-style": "^0.3.0",
+    "ember-computed-style": "joshbabier/ember-computed-style",
```

Comment:
Why did you need `jquery` in this addon's dev dependencies? I can only see it used in addon's tests.

Reply:
@lolmaus I wanted to make sure everything passed with the addon's tests